app/services/analytics_events.rb

```diff
@@ -648,6 +648,11 @@ def idv_gpo_verification_submitted(
     )
   end
 
+  # GPO verification visited
+  def idv_gpo_verification_visited
```

Comment:
this event used to have an older name via the "# Previously" comment, can we track that via tag?
```suggestion
  # @identity.idp.previous_event_name Account verification visited
  # GPO verification visited
  def idv_gpo_verification_visited
```